sdk/browser/src/AffinityWallet.ts

```diff
@@ -195,8 +195,7 @@ export class AffinityWallet extends CoreNetwork {
    * @returns array of ids for corelated records
    */
   async saveCredentials(data: any, storageRegion?: string): Promise<any> {
-    const encryptedCredentials = await this.walletStorageService.encryptCredentials(data)
-    const result = await this.saveEncryptedCredentials(encryptedCredentials, storageRegion)
+    const result = await this.walletStorageService.saveUnencryptedCredentials(data, storageRegion)
```

Comment:
@inga-affinidi this namespace is confusing. (make assumption that you will store credential in unEncrypted way). We could call it encryptAndStoreCredentials (if you want to have this as one method)

Reply:
the idea of why it's separate. Cause only encryption part is different depends on the platform (browser, react, expo), all other methods - its the same, so that's why we using here this separate platform related method for encryption/decryption, and then basic functionality inherited from core services.

Reply:
@DenisPopov15 , agree, I was hoping I'll be able to refactor all that stuff. Hopefully it won't take long and then it will make it to the non-draft version of that PR tomorrow. If not, I'll just rename the method.